n2v/inverter.py

```diff
@@ -275,6 +278,14 @@ def generate_components(self, guide_potential_components):
 
             self.va += v_fa
             self.vb += v_fa
+        elif "hartree" in guide_potential_components:
+            v_fa = (self.J0[0] + self.J0[1])
+
+            self.va += v_fa
+            self.vb += v_fa
+        else:
+            raise ValueError("Either Hartree or Fermi-Amaldi"
```

Comment:
Can we raise a warning instead of making it an error: "Hartee not FA was included... Convergence will likely not be achieved"? 
Not a good idea to do an inversion without any of this... but I think you should still be able to do it if you want.

Reply:
Make sense.